shared/config/happy_config.go

```diff
@@ -96,6 +96,7 @@ type HappyConfig struct {
 	data *ConfigData
 
 	envConfig *Environment
+	bootstrap *Bootstrap
```

Comment:
lets not add more to these structures, pass them into the function calls. These structures are getting too messy

Reply:
I think this one was missing from the happy config from the very beginning. This is essential contextual information that was never made available to happy. I am not in support of passing bootstrap references separate from HappyConfig, when HappyConfig is instantiated as `config.NewHappyConfig(bootstrapConfig)`; bootstrap has no meaning outside of HappyConfig.